python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py

```diff
@@ -637,6 +694,30 @@ async def configure(self, config):
             Script configuration, as defined by `schema`.
         """
 
+        if hasattr(config, "find_target") and hasattr(config, "track_target"):
+            raise RuntimeError(
+                "find_target and track_target configuration sections cannot be specified together."
+            )
+        elif hasattr(config, "find_target"):
+            self.log.debug(f"Finding target for cwfs @ {config.find_target}")
+            self.cwfs_target = await self.atcs.find_target(**config.find_target)
+            self.cwfs_target_ra = None
+            self.cwfs_target_dec = None
+            self.log.debug(f"Using target {self.cwfs_target} for cwfs.")
+        elif hasattr(config, "track_target"):
+            self.log.debug(f"Tracking target {config.track_target} for cwfs.")
+            self.cwfs_target = config.track_target.get("target_name", "cwfs_target")
+            self.cwfs_target_ra = None
+            self.cwfs_target_dec = None
+            if "icrs" in config.track_target:
+                self.cwfs_target_ra = config.track_target["icrs"]["ra"]
+                self.cwfs_target_dec = config.track_target["icrs"]["dec"]
+        else:
+            self.log.debug("No target configured.")
+            self.cwfs_target = None
+            self.cwfs_target_ra = None
+            self.cwfs_target_dec = None
+
```

Comment:
Is it worth putting a checkpoint here? 
I admit I'm a little lost on how checkpoints will be used in the future. Maybe after the slew is more appropriate.

Reply:
Can't have checkpoints in `configure`... but I'll add a checkpoint after it slew...